package.json

```diff
@@ -105,22 +107,29 @@
     "gl-pointcloud2d": "^1.0.3",
     "gl-scatter3d": "^1.2.3",
     "gl-select-box": "^1.0.4",
+    "gl-shader": "4.3.1",
     "gl-spikes2d": "^1.0.2",
     "gl-streamtube3d": "^1.4.1",
     "gl-surface3d": "^1.6.0",
     "gl-text": "^1.2.0",
     "glslify": "^7.1.1",
     "has-hover": "^1.0.1",
     "has-passive-events": "^1.0.0",
+    "incremental-convex-hull": "plotly/incremental-convex-hull.git#v1.1.0",
     "is-mobile": "^2.2.2",
     "mapbox-gl": "1.10.1",
     "matrix-camera-controller": "^2.1.4",
     "mouse-change": "^1.4.0",
     "mouse-event-offset": "^3.0.2",
     "mouse-wheel": "^1.2.0",
     "native-promise-only": "^0.8.1",
-    "ndarray": "^1.0.19",
+    "ndarray": "plotly/ndarray.git#v1.1.0",
```

Comment:
Looking at https://docs.npmjs.com/cli/v7/configuring-npm/package-json#github-urls it appears the `.git` can (should?) also be omitted.

Reply:
Done in https://github.com/plotly/plotly.js/pull/5888/commits/84b3a23cdcc0f4c45fcc48c5a1d24586ae6ded63.